### ISSUE TITLE
Bug: with --telemetry-enabled=false flag numary still sends data to twilio

### ISSUE THREAD

**Issue by quenbyako** [formancehq/ledger#368]
**Describe the bug**

No words guys:

```sh
$ numary --telemetry-enabled=false server start
  loading config file: Config File "numary" Not Found in "[/Users/r0ck3t/.numary /etc/numary]"
  segment 2022/11/15 23:58:59 ERROR: sending request - Post "https://api.segment.io/v1/batch": dial tcp: lookup api.segment.io: no such host
  segment 2022/11/15 23:58:59 ERROR: sending request - Post "https://api.segment.io/v1/batch": dial tcp: lookup api.segment.io: no such host
  segment 2022/11/15 23:58:59 ERROR: sending request - Post "https://api.segment.io/v1/batch": dial tcp: lookup api.segment.io: no such host
```

**To Reproduce**

1. Run `numary --telemetry-enabled=false server start`
2. See logs in firewall or in wireshark.

**Expected behavior**

Disable telemetry by default!

**Environment (please complete the following information):**

- MacOS Monterey, Macbook on M1
- Numary Version:

  ```
  Version: 1.8.0 
  Date: 2022-10-31T08:29:29Z 
  Commit: a55756b 
  ```

**Additional context**

You are doing something wrong guys, telemetry ***must*** be disabled by default in opensource apps, especially if they were build from source code. It's super stupid.

**Comment by gfyrag** [comment]
Hi @quenbyako.
We found the problem.
As a temporary workaround, add flag "--segment-enabled=false" (also "telemetry-enabled" flag), it should disable telemetry.

**Comment by jdupas22** [comment]
Hey @quenbyako ! The fix has been merged on main, it will be released on he 1.8.1 soon :) Thanks for pointing this out !
